keep-ui/entities/workflows/model/yaml.schema.ts

```diff
@@ -265,6 +279,7 @@ export function getYamlWorkflowDefinitionSchema(
     // TODO: move github and auth0 providers to the providers list from backend, once we have them at /providers endpoint
     githubStarsProvider,
     auth0LogsProvider,
+    fluxcdProvider,
```

Comment:
hey @Kunal-Darekar! this hack is only for providers with dots in names, like `github.stars`, for fluxcd provider, the schema will be automatically generated in the runtime for validation.
```suggestion
```